### ISSUE TITLE
navigation method

### ISSUE THREAD

**Issue by mpvanandel** [mpvanandel/paparazzi#10]
plan a route in the collision free space that optimizes distance coverage

**Comment by ErinTUDelft** [comment]
It could be that path planning (A*, tree search) are too expensive. There has been research on UAV which just uses an obstacle detector and sort of flies randomly while avoiding obstacles. We could use some very simple algorithm which balances conserving momentum with avoiding obstacles without expensive path planning, if needed.

**Comment by mpvanandel** [comment]
yeah that is also not my suggestion to do. My idea right now would be to make a convex region e.g. an ellipse and than follow this ellipse bounds with some offset. In terms of planning this should be really leight weight and lead to more covered distance than randomly moving around

**Comment by mpvanandel** [comment]
Only issue with a convex shape would be, that we cannot circle around an obstacle, so maybe we should consider other kinds of shapes too.

**Comment by mpvanandel** [comment]
But I would focus on navigation from wednesday onwards or maybe even friday. Before, that we need to finish the object detection and the depth perception, they are more pressing right now
